verilog/analysis/checkers/unpacked_dimensions_rule_test.cc

```diff
@@ -109,6 +109,19 @@ TEST(UnpackedDimensionsRuleTests, CheckRanges) {
       // inner range is packed dimensions, should not be detected as unpacked
       {"int some_array [bit [7:0]];"},
       {"int some_array [bit [0:7]];"},
+
+      // array of instances
+      {"module test (input logic blah);",
+       "endmodule",
+       "module tb;",
+       "  test test_i[3:0] (.blah (1'b0));",
+       "endmodule"},
+       
+       {"module test (input logic blah);",
+       "endmodule",
+       "module tb;",
+       "  test test_i[0:3] (.blah (1'b0));",
+       "endmodule"}
```

Comment:
Just noticed, you can omit the commas between joined string literals.  But you might want to place a space or \n between "endmodule" and "module" or they'll be fused together.

Just make sure `bazel test //...` works.  :)

Reply:
Thanks for feedback. I've seen tests written in both ways, is there a preferred one?

Reply:
I would only separate with a `,` when marking an expected finding, like `{"good text", {kFoo, "bad-text"}, "more good text"}`

Reply:
OK. I will take this into account next time. Thanks.